ed448/rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+newline_style = "Unix"
```

Comment:
probably drop this file, to align with the rest of the repo

Reply:
Dropped!